openmdao/docs/usr-guide/tutorials/recording.rst

```diff
@@ -874,7 +874,7 @@ coordinate string descriptor, or as a standard python index.
    :options: -ELLIPSIS, +NORMALIZE_WHITESPACE
 
     Optimization terminated successfully.    (Exit mode 0)
-                Current function value: [-27.33333333]
+                Current function value: -27.3333333333
```

Comment:
Not sure what to do about this, but that is really annoying that the new scipy is like that. Maybe we should just disable the doctest on these output blocks for now. There are ways in the doctest plugin to markup the output so that it can pass with an optional extra character there, but I think it impacts readability of the doc.